src/server/game/Spells/Spell.cpp

```diff
@@ -5400,13 +5394,13 @@ SpellCastResult Spell::CheckCast(bool strict, uint32* param1 /*= nullptr*/, uint
                 if (res != SPELL_CAST_OK)
                     return res;
 
-                // chance for fail at orange mining/herb/LockPicking gathering attempt
+                // chance for fail at lockpicking attempt
                 // second check prevent fail at rechecks
                 if (skillId != SKILL_NONE && (!m_selfContainer || ((*m_selfContainer) != this)))
                 {
-                    bool canFailAtMax = skillId != SKILL_HERBALISM && skillId != SKILL_MINING;
+                    bool canFailAtMax = skillId == SKILL_LOCKPICKING;
```

Comment:
What's the purpose of this change? What `skillId`s other than `_HERBALISM`, `_MINING` and `_LOCKPICKING` does this affect?

Reply:
Easier to understand for readers.

Reply:
Easier to understand for future readers

Reply:
Ah, so there's no other skill IDs? Alright.